automatic/glmixer/glmixer.nuspec

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>glmixer</id>
+    <version>0.0</version>
+    <title>Graphic Live Mixer</title>
+    <owners>chocolatey</owners>
+    <authors>Bruno Herbelin</authors>
+    <projectUrl>https://glmixer.sourceforge.net/</projectUrl>
```

Comment:
this page no longer exists it seems. please change it to point to the correct location